lib/ansible/parsing/vault/__init__.py

```diff
@@ -319,10 +319,9 @@ def bytes(self):
         return None
 
     def load(self):
-        self._bytes = self.read_file(self.filename, self.loader)
+        self._bytes = self.read_file(self.filename)
 
-    @staticmethod
-    def read_file(filename, loader):
+    def read_file(self, filename):
```

Comment:
Nothing uses the read_file() statically so it doesn't gain anything other to make it static in one class and regular in another. I'd prefer them to be consistent.

But looking at it closer, it also doesnt need to be a public method at all. It is just a helper implementation of the the more abstract load().

Can probably make FileVaultSecret.read_file() use self.loader now (or possibly with
all or part of https://github.com/ansible/ansible/pull/27668/files and https://github.com/ansible/ansible/pull/27669/). So vault password files could use loader for encrypted files and whatever else dataloader enables.